packages/pkl.impl.ghactions/PklCI.pkl

```diff
@@ -471,6 +478,19 @@ local function withTriggerWorkflows(triggerType: String): Mixin<Workflow.Jobs> =
   }
 }
 
+local withHawkeye: Mixin<Workflow.Jobs> = (it) -> (it) {
+  when (hawkeyeCheck != null) {
+    for (key in it.keys) {
+      when (key.startsWith("deploy")) {
+        [key] {
+          needs { hawkeyeCheck.jobName }
+        }
+      }
```

Comment:
I like that we're adding a hawkeye job definition, but I think this logic here is assuming a little too much about how workflows are set up, and how jobs are named (e.g. you must call your deploy jobs "deploy-*")

I think we can just provide a hawkeye job, and then wire it up per repo that needs it.

The output converter can be based on the job name:

```pkl
    [[true]] {
      renderer {
        converters {
          [Workflow.DefaultJob] = (it) ->
              if (it.name == "hawkeye-check")
                it.toMap().put("container", Map("image", hawkeyeCheck.image))
              else it
        }
      }
    }
```

And then we'd just add a `name = "hawkeye-check"` to the hawkeye check job.